core/lib/spree/core/importer/order.rb

```diff
@@ -100,74 +100,25 @@ def self.create_shipments_from_params(shipments_hash, order)
-                    "2": { "variant_id": 123, "quantity": 1 }
-                  }
-                }
-              }
-            EOS
-
-            line_items.each_key do |k|
-              begin
-                adjustments = line_items[k].delete(:adjustments_attributes)
-                extra_params = line_items[k].except(:variant_id, :quantity, :sku)
-                line_item = ensure_variant_id_from_params(line_items[k])
-                variant = Spree::Variant.find(line_item[:variant_id])
-                line_item = order.contents.add(variant, line_item[:quantity])
-                # Raise any errors with saving to prevent import succeeding with line items
-                # failing silently.
-                if extra_params.present?
-                  line_item.update_attributes!(extra_params)
-                else
-                  line_item.save!
-                end
-                create_adjustments_from_params(adjustments, order, line_item)
-              rescue Exception => e
-                raise "Order import line items: #{e.message} #{line_item}"
-              end
-            end
-          when Array
-            line_items.each do |line_item|
-              begin
-                adjustments = line_item.delete(:adjustments_attributes)
-                extra_params = line_item.except(:variant_id, :quantity, :sku)
-                line_item = ensure_variant_id_from_params(line_item)
-                variant = Spree::Variant.find(line_item[:variant_id])
-                line_item = order.contents.add(variant, line_item[:quantity])
-                # Raise any errors with saving to prevent import succeeding with line items
-                # failing silently.
-                if extra_params.present?
-                  line_item.update_attributes!(extra_params)
-                else
-                  line_item.save!
-                end
-                create_adjustments_from_params(adjustments, order, line_item)
-              rescue Exception => e
-                raise "Order import line items: #{e.message} #{line_item}"
+          return [] unless line_items
+
+          line_items.each do |line_item|
+            begin
+              adjustments = line_item.delete(:adjustments_attributes)
+              extra_params = line_item.except(:variant_id, :quantity, :sku)
+              line_item = ensure_variant_id_from_params(line_item)
+              variant = Spree::Variant.find(line_item[:variant_id])
+              line_item = order.contents.add(variant, line_item[:quantity])
+              # Raise any errors with saving to prevent import succeeding with line items
+              # failing silently.
+              if extra_params.present?
+                line_item.update_attributes!(extra_params)
+              else
+                line_item.save!
               end
+              create_adjustments_from_params(adjustments, order, line_item)
+            rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?